src/inputs/schemas/music/musicAlbum0.json

```diff
@@ -67,13 +67,12 @@
       "name": "Language",
       "description": "The language of the song lyrics in the album.",
       "required": false,
-      "type": "InternalVec",
-      "maxItems": 5,
+      "type": "Internal",
```

Comment:
when you specify Internal  you also need to specify the class Id of the internal type ?

Reply:
ah its there on next line :D